cmd/srcd/cmd/init_test.go

```diff
@@ -200,3 +197,104 @@ func (s *InitTestSuite) TestWithRunningOtherComponents() {
 		require.Equal(exp, actualMsg[i])
 	}
 }
+
+func (s *InitTestSuite) initGitRepo(path string) {
+	s.T().Helper()
+
+	err := os.MkdirAll(path, os.ModePerm)
+	s.Require().NoError(err)
+
+	cmd := exec.Command("git", "init", path)
```

Comment:
should we ensure someway that `git` is already available?

Reply:
If `git` is not installed the tests will complain in the next `s.Require().NoError(err)`.
What else did you have in mind?

Reply:
maybe as a precondition for the `test-integration` make target...
but not a strong opinion.

Reply:
I understand your point, but I'd prefer to keep it simple, at least for now. We could check many things (docker is installed, etc), but in the end the tests are meant for our CI, that we control, and the developers machines. So to me it's safe to assume some tools.

Reply:
is the computer on? do we have network? :dancer: